assignments/objects.js

```diff
@@ -18,27 +18,70 @@ const example = {
 }
 
 // Write your intern objects here:
-
-
+const InternList = [
+{
+  "id": 1,
+  "name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"
+},
+ {
+  "id": 2,
+  "name": "Kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M"
+},
+
+{
+  "id": 3,
+  "name": "Keven",
+  "email": "kmummery2@wikimedia.org",
+  "gender": "M"
+},
+
+{
+  "id": 4,
+  "name": "Gannie",
+  "email": "gmartinson3@illinois.edu",
+  "gender": "M"
+},
+{
+  "id": 5,
+  "name": "Antonietta",
+  "email": "adaine5@samsung.com",
+  "gender": "F"
+}
+];
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
+console.log(InternList[0].name);
 
 // Kennan's ID
+console.log(InternList[1].id);
 
 // Keven's email
+console.log(InternList[2].email);
 
 // Gannie's name
+console.log(InternList[3].name);
 
 // Antonietta's Gender
+console.log(InternList[4].gender);
 
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
-// console.log(kennan.speak());
+
+const speak = "Hello, my name is" + this.name;
```

Comment:
the way you set up the object methods is not going to work.  

1. What is happening is that you instantiated the InternList with no methods
2. then you instantiated a speak function separately, 
3. then you tried access the `speak` method of intern[4].

The problem is that intern[4] does not have such a method yet.  You would need to first assign that Method to intern[4] before you can use it.

so after step2, `InternList[4].speak = speak` that means you assigned a `speak` key/method to InternList[4] with value of your speak function.

Then you would be able to call it like you did.